dbeaver/dbeaver
# #35475 Fix negative curJobRunning value

### How to test

1. Create procedure with pg_sleep(10000)
```
create or replace procedure proc1()
language plpgsql
AS 
$$
begin
	select pg_sleep(10000);
end;
$$
;
```
2. Introduce query `call proc1();` to the script
3. Select this query and execute
4. Press disconnect and agree to stop query execution
5. Select this query and execute
6. Check the state of the `Execute-->Cancel active query` context menu item. It should be enabled while the query execution task is running (you can check it in Background Tasks view)
